bread/bread.py

```diff
@@ -201,6 +221,41 @@ def get_context_data(self, **kwargs):
                     data['previous_url'] = self._get_new_url(page=page.previous_page_number())
         return data
 
+    # The following is copied from the Django admin and tweaked
+    def get_search_results(self, request, queryset, search_term):
+        """
+        Returns a tuple containing a queryset to implement the search,
+        and a boolean indicating if the results may contain duplicates.
+        """
+        # Apply keyword searches.
+        def construct_search(field_name):
+            if field_name.startswith('^'):
+                return "%s__istartswith" % field_name[1:]
+            elif field_name.startswith('='):
+                return "%s__iexact" % field_name[1:]
+            elif field_name.startswith('@'):
+                return "%s__search" % field_name[1:]
+            else:
+                return "%s__icontains" % field_name
+
+        use_distinct = False
+        search_fields = self.search_fields
+        if search_fields and search_term:
+            orm_lookups = [construct_search(str(search_field))
+                           for search_field in search_fields]
+            for bit in search_term.split():
+                or_queries = [Q(**{orm_lookup: bit})
+                              for orm_lookup in orm_lookups]
+                queryset = queryset.filter(reduce(or_, or_queries))
+            if not use_distinct:
```

Comment:
I don't see how `use_distinct` would be anything but False at this point. Is the check necessary?